### PR TITLE
Issue #101 resolved

### PR DESCRIPTION
**Pull request by shraddha-1402**
## [Frontend] Fix the carousel section of homepage

 Closes: #101 

## Proposed Changes
1. Fixed the height of carousel
2. Made it responsive

### Brief description of what is fixed or changed 
Fixed height of carousel to smaller sized posters content
Made changes to CSS by adding media queries
Added responsiveClass value to true in Owl Carousel options

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
Now:
image
image
image

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: frontend/src/components/home/carousel/carousel.css [hunk 1]
```diff
@@ -58,3 +61,25 @@ span {
...
+    height: auto;
+    width: 90%;
+    padding-left: 10%;
+  }
+}
+@media(min-width: 800px){
+  .owl-carousel .owl-item img{
+    width: 90%;
+    padding-left: 10%;
+    /* padding-right: 5%; */
```
Comment (Kajol-Kumari) [inline frontend/src/components/home/carousel/carousel.css:78]: please remove the commented lines

File: frontend/src/components/home/carousel/carousel.css [hunk 2]
```diff
@@ -58,3 +61,25 @@ span {
 .owl-nav button:focus {
   outline: none;
 }
+@media(max-width: 500px){
```
Comment (Kajol-Kumari) [inline frontend/src/components/home/carousel/carousel.css:64]: leave one blank line between all two css classes

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Please make the suggested formatting and fix the left and right arrow key of the carousel
Screenshot from 2020-12-30 13-22-04

File: frontend/src/components/home/carousel/carousel.css (hunk 2 above)
Comment (shraddha-1402) [reply frontend/src/components/home/carousel/carousel.css:64]: On it!

Comment (shraddha-1402) [reply frontend/src/components/home/carousel/carousel.css:64]: image

File: frontend/src/components/home/carousel/carousel.css (hunk 1 above)
Comment (shraddha-1402) [reply frontend/src/components/home/carousel/carousel.css:78]: Removed all comments.

**Comment by shraddha-1402** [comment]
> Please make the suggested formatting and fix the left and right arrow key of the carousel
> Screenshot from 2020-12-30 13-22-04

Please check if this resolves the issue

https://user-images.githubusercontent.com/69422250/103341221-4fe26700-4aac-11eb-9632-ff9df646716e.mp4

**Comment by Kajol-Kumari** [comment]
Hey @shraddha-1402 the left and right arrow key is still looking same when I am trying to run it on my local. Please check if it's appearing correctly on your end.

**Comment by shraddha-1402** [comment]
> Hey @shraddha-1402 the left and right arrow key is still looking same when I am trying to run it on my local. Please check if it's appearing correctly on your end.

It is running perfectly fine at my end, i sent you the video showing the carousel for different screen sizes.

**Comment by shraddha-1402** [comment]
> > Please make the suggested formatting and fix the left and right arrow key of the carousel
> > Screenshot from 2020-12-30 13-22-04
> 
> Please check if this resolves the issue
> 
>  2020-12-30.14-22-31.1.mp4

@Kajol-Kumari Please check this video

**Comment by shraddha-1402** [comment]
> > Please make the suggested formatting and fix the left and right arrow key of the carousel
> > Screenshot from 2020-12-30 13-22-04
> 
> Please check if this resolves the issue
> 
>  2020-12-30.14-22-31.1.mp4

Please check the video linked here.

**Comment by Kajol-Kumari** [comment]
@Abhishek971999 can you please test it once and check if it's showing up in the correct way

**Comment by Abhishek971999** [comment]
> @Abhishek971999 can you please test it once and check if it's showing up in the correct way

Yes @Kajol-Kumari I am having the same issue . 
Capture

@shraddha-1402 Please check again with this bug

**Comment by shraddha-1402** [comment]
> > @Abhishek971999 can you please test it once and check if it's showing up in the correct way
> 
> Yes @Kajol-Kumari I am having the same issue .
> Capture
> 
> @shraddha-1402 Please check again with this bug

I checked everything its running perfectly in my pc, could this be the problem of browers? which one are you using?

**Comment by Kajol-Kumari** [comment]
Hey @shraddha-1402 I am using chrome for testing this

**Comment by shraddha-1402** [comment]
@Kajol-Kumari  @Abhishek971999 
My friend tried this on his system and the carousel works perfectly fine
```
git init
git remote -v
git remote add origin https://github.com/HITK-TECH-Community/Community-Website.git
git remote -v
git fetch origin e0c0b2b0b52ec40d662e0e509d38d9e9f07435e0      
git reset --hard FETCH_HEAD
cd frontend/
npm install
npm start
```
Please try this on your system.
I am not able to recreate this issue, any suggestions?

**Comment by shraddha-1402** [comment]
Hey @Kajol-Kumari I am deleting this pr coz i accidentally committed changes of issue #99 in the same branch. So i'll send another pr, please consider that one. sorry for inconvenience.

**Pull request closed without merging**